tests/cron_job_tests/test_genomic_pipeline.py

```diff
@@ -1789,7 +1789,7 @@ def test_gem_a1_manifest_end_to_end(self):
             self.assertEqual("yes", rows[0]['consent_for_ror'])
             self.assertEqual(test_member_1.consentForGenomicsRORAuthored, parse(rows[0]['date_of_consent_for_ror']))
             self.assertEqual(test_member_1.chipwellbarcode, rows[0]['chipwellbarcode'])
-            self.assertEqual(test_member_1.gcSiteId, rows[0]['genome_center'])
+            self.assertEqual('JH', rows[0]['genome_center'])
```

Comment:
Was this an intended change to a hardcoded string, rather than something like test_member_1.gcSiteId.upper()?

Reply:
Yes, this was intended. Thanks for checking!